doc/ref/configuration/master.rst

```diff
@@ -2599,6 +2599,8 @@ can have multiple root directories. The subdirectories in the multiple file
 roots cannot match, otherwise the downloaded files will not be able to be
 reliably ensured. A base environment is required to house the top file.
 
+As of 2018.3.4, it is possible to have `__env__` as a catch-all environment.
```

Comment:
Can you change this to "2018.3.5 and 2019.2.1"? This missed the branching for both 2018.3.4 and 2019.2.0. Other than that I have no problem merging it.